modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -1027,20 +1045,30 @@ impl Wallet {
         our_peer_id: PeerId,
         module_api: DynModuleApi,
     ) -> Result<Wallet, WalletCreationError> {
-        Self::spawn_broadcast_pending_task(task_group, &bitcoind, db);
+        let propose_citem = Arc::new(Notify::new());
+        let (block_count_tx, block_count_rx) = watch::channel(None);
+        let (fee_rate_tx, fee_rate_rx) = watch::channel(cfg.consensus.default_fee);
 
-        let fee_rate_rx = bitcoind
+        Self::spawn_broadcast_pending_task(task_group, &bitcoind, db);
+        bitcoind
             .clone()
-            .spawn_fee_rate_update_task(
-                task_group,
-                cfg.consensus.default_fee,
-                cfg.consensus.network.0,
-                CONFIRMATION_TARGET,
-            )
+            .spawn_fee_rate_update_task(task_group, cfg.consensus.network.0, CONFIRMATION_TARGET, {
+                let propose_citem = propose_citem.clone();
+                move |feerate| {
+                    let _ = fee_rate_tx.send(feerate);
+                    propose_citem.notify_one();
```

Comment:
Should we check if anything changed a all?

Reply:
Generaly with `Notify` if you ever suspect something changed, might fire them needlessly anyway.